### DISCUSSION TITLE
Welcome to solidjs-the-complete-guide Discussions!

### DISCUSSION THREAD

**Discussion by snnsnn** [solid-courses/solidjs-the-complete-guide#5] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by everton-dgn** [comment]
Suggestion: Publish the book in HTML/Markdown for easier translation

Hi, and thanks for this amazing resource!

I’m from Brazil, and my native language is Portuguese. I’d like to suggest publishing the book in Markdown (.md) or HTML format to make it more accessible for non-English speakers. These formats would allow:
	•	Easier translation with tools like Google Translate or DeepL.
	•	Community contributions to help translate parts of the book.
	•	More flexibility in reading across different devices.

Would this be something you’d consider? It could really help expand the reach of SolidJS to more developers worldwide.

Thanks! 😊

**Reply by snnsnn** [reply]
Hi, Éverton. I’d be happy to. I’m planning to offer translations in major languages, but I have a few priorities:

1. Add missing parts (chapters on Router and SolidStart).
 2. Add hands-on examples.
 3. Provide a GitHub repository for the examples.

But supporting more formats means extra work. Let me finalize the book first and see what I can do. Doing this for something that’s still evolving also adds to the workload.

The first two are almost complete. The third one depends on the book reaching a more stable state. Right now, there’s a minor flux as I make minor rewrites based on user feedback.

Once the book is finalized, I’ll provide a way to read translated text and suggest corrections or alternatives. This will help coordinate efforts so that different people aren’t duplicating the same work independently.

Regards.
